### PR TITLE
refactor,chore(egen) : refactored code as per template guidelines , performed linter test

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes 
 1.changed REGION to LOCATION
 2.removed os module
 3.removed os.env(IS_TESTING) in cleanup section
 4. performed linter test
 5. followed template guidelines

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb [hunk 1]
```diff
@@ -444,7 +367,7 @@
         "id": "f8fd397cc4f6"
       },
       "source": [
-        "### Download the Dataset to Cloud Storage"
+        "# Download the Dataset to cloud storage"
```
Comment (kittyabs) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:370]: Should be "Cloud Storage" (It's a product name)

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb [hunk 2]
```diff
@@ -580,7 +503,7 @@
         "id": "k_QorXXztzPH"
       },
       "source": [
-        "Start a new experiment run to track training parameters and start the training job. Note that this operation will take around 10 mins."
+        "Start a new experiment run to track training parameters and start the training job. Note that this operation  take around 10 mins."
```
Comment (kittyabs) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:506]: remove extra space and change "take" to "takes" (to agree with the singular subject):

- From: Note that this operation  take around 10 mins.
- To: Note that this operation takes around 10 mins.

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb [hunk 3]
```diff
@@ -620,7 +543,7 @@
         "id": "O-uCOL3Naap4"
       },
       "source": [
-        "Next, deploy your Vertex AI Model resource to a Vertex AI Endpoint resource. This operation will take 10-20 mins."
+        "Next, deploy your Vertex AI Model resource to a Vertex AI Endpoint resource. This operation  take 10-20 mins."
```
Comment (kittyabs) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:546]: - lower case "endpoint":
    - Vertex AI endpoint
- remove extra space and change "take" to "takes" and "mins" to "minutes"
    - This operation takes 10-20 minutes

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb [hunk 4]
```diff
@@ -824,15 +747,15 @@
       },
       "outputs": [],
       "source": [
-        "# Warning: Setting this to true will delete everything in your bucket\n",
+        "# Warning: Setting this to true  delete everything in your bucket\n",
```
Comment (kittyabs) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:750]: remove extra space and and change "delete" to "deletes":

- Warning: Setting this to true deletes everything in your bucket

**Review by kittyabs — COMMENTED** [review]
several needed edits

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb [hunk 5]
```diff
@@ -580,7 +503,7 @@
         "id": "k_QorXXztzPH"
       },
       "source": [
-        "Start a new experiment run to track training parameters and start the training job. Note that this operation will take around 10 mins."
+        "Start a new experiment run to track training parameters and start the training job. Note that this operation takes around 10 mins."
```
Comment (kittyabs) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:506]: change "mins" to "minutes"

**Review by kittyabs — COMMENTED** [review]
small edit

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb [hunk 6]
```diff
@@ -141,162 +143,85 @@
...
-      },
-      "outputs": [],
-      "source": [
-        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "    import IPython\n",
         "\n",
-        "# Set the project id\n",
-        "! gcloud config set project {PROJECT_ID}"
+        "    app = IPython.Application.instance()\n",
+        "    app.kernel.do_shutdown(True)"
```
Comment (katiemn) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:175]: Please add the warning message from the notebook template

**Review by katiemn — APPROVED** [review]

**Pull request merged**